packages/calcite-components/src/components/color-picker/color-picker.scss

```diff
@@ -3,6 +3,18 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-color-picker-save-buttons-background-color-active: defines the background color of the button when in a active state in the component.
```

Comment:
Not critical for this PR since it can be tackled when `color-picker` gets updated, but `save-buttons` in the name could be misleading. Is there another name we could use to describe the saved color section, which contains both add/remove color buttons?

Reply:
I previously had it named `--calcite-color-picker-button-background-color-active` but was told that sounded to generic but I think that might be the right option here since add/remove are the only buttons in the component.